model-archiver/model_archiver/model_packaging_utils.py

```diff
@@ -25,6 +25,7 @@
 
 
 model_handlers = {
+        'base_handler': 'tables',
```

Comment:
was this change tested? What's the intent?

Reply:
There is only one place in the code where `model_handlers` is used.

The handler can either be a built-in handler (not copied to the model archive) or must be a python file (copied to the model archive). The behavior is decided based on the presence of the handler name in that dictionary.

It's only possible to use the base handler as a built-in handler. (Using the `base_handler.py` file does not work since that file uses relative imports that break when the file if copied to the model archive).

So, to make it possible to use the base_handler in model archive creator is to add it to the it to the `model_handlers` dict.

Reply:
Hmm I've been wanting to remove all relative imports in our handler so maybe this a good excuse to do so

Reply:
I think this PR would still be beneficial.
It's much easier to specify a built-in handler name vs putting a file inside a container.
It's really surprising to me that TorchServe supports serving advanced models out of the box (images, text), but does not support the basics (tabular models).

What concerns do you have about adding the `base_handler` to that dictionary?